src/options.js

```diff
@@ -111,4 +114,6 @@ export function getSerializerOptions({
     singlePass,
     trace,
   };
+  if (additionalFunctions) result.additionalFunctions = additionalFunctions;
```

Comment:
additionalFunctions, like all other option properties, is optional. If present, however, its value must be an array. This is the only way I could find that would satisfy these two conditions at once.